src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/security/AuthSource.kt

```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.digital.hmpps.externalusersapi.security
+
+import com.fasterxml.jackson.annotation.JsonValue
+
+enum class AuthSource(val description: String) {
```

Comment:
As mentioned above this enum feels unnecessary within external-users-api.

Reply:
Will take this clean up exercise after DEVs discussion tomorrow